src/components/renderer/add-loop-row.vue

```diff
@@ -20,14 +20,18 @@
 </template>
 
 <script>
+import { mapActions } from "vuex";
+
 export default {
   props: {
     value: Array,
     config: Object,
     error: String,
   },
   methods: {
-    add() {
+    ...mapActions("globalErrorsModule", ["unlocked"]),
```

Comment:
Can you change the function name from `unlocked` to something more relevant? Is hard to tell from the PR what the function is actually doing, or even reading from it in the code.

Reply:
Seems this function is not required any more, I will review to remove this and the related ones